indico/modules/events/registration/controllers/management/sections.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import request, jsonify
+
+from indico.core.db import db
+from indico.modules.events.registration.controllers.management import RHManageRegFormBase
+from indico.modules.events.registration.models.items import RegistrationFormSection
+from indico.modules.events.registration.models.registration_form_fields import (RegistrationFormField,
+                                                                                RegistrationFormFieldData)
+
+
+class RHManageRegFormSectionBase(RHManageRegFormBase):
+    CSRF_ENABLED = False
+
+    normalize_url_spec = {
+        'locators': {
+            lambda self: self.section
+        }
+    }
+
+    def _checkParams(self, params):
+        RHManageRegFormBase._checkParams(self, params)
+        self.section = RegistrationFormSection.get_one(request.view_args['section_id'])
+
+
+class RHRegFormAddSection(RHManageRegFormBase):
+    CSRF_ENABLED = False
+
+    def _process(self):
+        section = RegistrationFormSection(registration_form=self.regform)
+        section.title = request.json['title']
+        section.description = request.json.get('description', '')
```

Comment:
is it actually missing from the json data when empty? otherwise `['description']` should be fine